### ISSUE TITLE
ImportError: no module named examples

### ISSUE THREAD

**Issue by cdagnino** [pymc-devs/pymc3#573]
I installed PyMC 3.0 with pip install on July 20th, 2014.
I ran pymc.test()   and everything is passed except I got five errors, all resulting from

`from pymc.examples import glm_linear, glm_robust` and the like triggering
`ImportError: No module named examples` 

or

`from pymc import examples`   triggering
`ImportError: cannot import name examples`

**Comment by fonnesbeck** [comment]
Thanks for the report. Can you install again and post the full output? It looks like something went wrong on this installation.

**Comment by kyleam** [comment]
Isn't this the same issue as #531?  It doesn't look like the fix over there was merged.

**Comment by cdagnino** [comment]
@fonnesbeck : I pip uninstalled and then installed with the following command once again:

`pip install --no-deps git+git://github.com/pymc-devs/pymc.git`
I can try a different `pip install`if you think that's the issue.

Here is the result for `pymc.test('full')`

```
//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/step_methods/metropolis.py:44: DeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  return np.random.multivariate_normal(mean=np.zeros(self.s.shape[0]), cov=self.s)
//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/step_methods/metropolis.py:44: DeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  return np.random.multivariate_normal(mean=np.zeros(self.s.shape[0]), cov=self.s)
 [-----------------100%-----------------] 20000 of 20000 complete in 12.6 sec....................
======================================================================
ERROR: Failure: ImportError (No module named examples)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/tests/test_diagnostics.py", line 3, in <module>
    from pymc.examples import disaster_model as dm
ImportError: No module named examples

======================================================================
ERROR: Failure: ImportError (cannot import name examples)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/tests/test_examples.py", line 2, in <module>
    from pymc import examples
ImportError: cannot import name examples

======================================================================
ERROR: Failure: ImportError (No module named examples)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/tests/test_glm.py", line 11, in <module>
    from pymc.examples import glm_linear, glm_robust
ImportError: No module named examples

======================================================================
ERROR: pymc.tests.test_plots.test_plots
----------------------------------------------------------------------
Traceback (most recent call last):
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/tests/test_plots.py", line 15, in test_plots
    from pymc.examples import arbitrary_stochastic as asmod
ImportError: No module named examples

======================================================================
ERROR: pymc.tests.test_plots.test_multichain_plots
----------------------------------------------------------------------
Traceback (most recent call last):
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/tests/test_plots.py", line 48, in test_multichain_plots
    from pymc.examples import disaster_model as dm
ImportError: No module named examples

----------------------------------------------------------------------
Ran 250 tests in 598.941s

FAILED (errors=5)
Out[2]: <nose.result.TextTestResult run=250 errors=5 failures=0>
```

**Comment by fonnesbeck** [comment]
@kyleam You are correct! I have merged this now. Please test.

**Comment by cdagnino** [comment]
@fonnesbeck : I reinstalled. Now it throws an error when importing pymc (not sure if it's an issue with my installation)

```
In [1]: import pymc
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-5f262cfcb99b> in <module>()
----> 1 import pymc

//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/__init__.py in <module>()
      6
      7 from .stats import summary
----> 8 from .sampling import *
      9 from .step_methods import *
     10 from .tuning import *

//anaconda/envs/pymc_works/lib/python2.7/site-packages/pymc/sampling.py in <module>()
----> 1 from pymc import backends
      2 from pymc.backends.base import merge_traces, BaseTrace, MultiTrace
      3 from pymc.backends.ndarray import NDArray
      4 import multiprocessing as mp
      5 from time import time

ImportError: cannot import name backends
```

**Comment by kyleam** [comment]
This should be fixed on the current master (fb28d87).

**Comment by cdagnino** [comment]
@kyleam , @fonnesbeck : 
It is passing `pymc.test()` in my computer now, thanks for following up on this!
I'll close this issue if that's OK with you.
